src/main/java/redis/clients/jedis/params/GeoRadiusStoreParam.java

```diff
@@ -0,0 +1,112 @@
+package redis.clients.jedis.params;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+import redis.clients.jedis.util.SafeEncoder;
+
+public class GeoRadiusStoreParam extends Params {
+    private static final String STORE = "store";
+    private static final String STOREDIST = "storedist";
+
+    public GeoRadiusStoreParam() {
+    }
+
+    public static GeoRadiusStoreParam geoRadiusStoreParam() {
+        return new GeoRadiusStoreParam();
+    }
+
+    public GeoRadiusStoreParam store(String key) {
+        if (key != null) {
+            addParam(STORE, key);
```

Comment:
why do you need it to be Params? it can only be one of the two options.
Params seems like a waste here.

Reply:
@gkorland It was actually my idea. You can check alternate implementation #2140